src/settings/SettingsPage.js

```diff
@@ -31,6 +32,19 @@ export const SettingsPage = ({
       <title>{t('title')} - IPFS</title>
     </Helmet>
 
+    <Box className='mb3 pa4'>
+      <Title>{t('pinningServices.title')}</Title>
+      <Trans i18nKey='pinningServices.description'>
+        <p className='ma0 mr2 lh-copy charcoal f6'>
+          <span>Use local pinning to ensure files on your local node persist and are never garbage-collected.
+          You can also link your accounts with other pinning services to automatically or selectively persist files with those providers, enabling you to keep backup copies of your files and/or make them available to others when your local node is offline. </span>
```

Comment:
```suggestion
          You can also link your accounts with other remote pinning services to automatically or selectively persist files with those providers, enabling you to keep backup copies of your files and/or make them available to others when your local node is offline. </span>
```
My bad on this one, the language has evolved somewhat 😊

Reply:
Done 😄